source/frontend/src/features/mapSideBar/acquisition/AcquisitionView.tsx

```diff
@@ -158,6 +162,12 @@ export const AcquisitionView: React.FunctionComponent<IAcquisitionViewProps> = (
             }
             bodyComponent={
               <StyledFormWrapper>
+                {error && (
+                  <b>
+                    Failed to load Acquisition File. Check the detailed error in the top right for
+                    more details.
+                  </b>
+                )}
```

Comment:
nit: please add unit tests to cover this new logic
![image](https://github.com/bcgov/PSP/assets/24322224/4f3c83fe-a3da-42f1-a044-c40c1851e368)